Cleaning Code/cleaning_traffic.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[3]:
+
+
+import csv
+import pandas as pd
+import numpy as np
+
+
+# In[33]:
+
+
+traffic_df_old = pd.read_csv("Chicago_Traffic_2011-2018.csv", nrows = 2000)
```

Comment:
It would be nice to change the name of this variable. Having the word "old" in it but still adding new column names to it is confusing. Maybe just traffic_df could be good